modules/installation/proc-add-custom-app-config-file-ocp-operator.adoc

```diff
@@ -1,99 +0,0 @@
-[id="proc-add-custom-app-config-file-ocp-operator_{context}"]
-= Adding a custom application configuration file to {ocp-short} using the Operator
-
-A custom application configuration file is a `ConfigMap` object that you can use to change the configuration of your {product} instance. If you are deploying your {product-short} instance on {ocp-brand-name}, you can use the {product} Operator to add a custom application configuration file to your {ocp-short} instance by creating the `ConfigMap` object and referencing it in the {product-short} custom resource (CR).
-
-The custom application configuration file contains a sensitive environment variable, named `BACKEND_SECRET`. This variable contains a mandatory backend authentication key that {product-short} uses to reference an environment variable defined in an {ocp-short} secret. You must create a secret, named 'secrets-rhdh', and reference it in the {product-short} CR.
```

Comment:
What was this and above paragraph removed? I don't see them anywhere anymore

Reply:
The info about config maps moved to introductions.

The info about the backend secret moved to step 1 in _Provisioning your custom {product} configuration_.